app/ide-desktop/lib/client/src/file-associations.ts

```diff
@@ -101,7 +101,7 @@ export function isFileOpenable(path: string): boolean {
  * it receives the `open-file` event (and no new instance is created for us). In this case,
  * we manually start a new instance of the application and pass the file path to it (using the
  * Windows-style command). */
-export function onFileOpened(event: Event, path: string): string | null {
```

Comment:
This is needed, otherwise the linter fails with error 
```
INFO ide_ci::program::command: npmℹ️ src/file-associations.ts(144,40): error TS2345: Argument of type '{ preventDefault: () => void; readonly defaultPrevented: boolean; }' is not assignable to parameter of type 'Event'.
INFO ide_ci::program::command: npmℹ️   Type '{ preventDefault: () => void; readonly defaultPrevented: boolean; }' is missing the following properties from type 'Event': bubbles, cancelBubble, cancelable, composed, and 16 more.
```
It seems the `open-file` event is not of type Event, even if electron apps docs says so: https://www.electronjs.org/docs/latest/api/app#event-open-file-macos

This is what I found working.